airbyte-integrations/connectors/destination-s3/src/main/java/io/airbyte/integrations/destination/s3/S3ConsumerFactory.java

```diff
@@ -80,26 +80,19 @@ public AirbyteMessageConsumer create(final Consumer<AirbyteMessage> outputRecord
       final AirbyteStream abStream = stream.getStream();
       final String namespace = abStream.getNamespace();
       final String streamName = abStream.getName();
-      final String outputNamespace = getOutputNamespace(abStream, config.get(BUCKET_PATH_FIELD).asText(), namingResolver);
-      final String customOutputFormat =
-          config.has(PATH_FORMAT_FIELD) && !config.get(PATH_FORMAT_FIELD).asText().isBlank() ? config.get(PATH_FORMAT_FIELD).asText()
-              : S3DestinationConstants.DEFAULT_PATH_FORMAT;
-      final String outputBucketPath = storageOperations.getBucketObjectPath(outputNamespace, streamName, SYNC_DATETIME, customOutputFormat);
+      final String bucketPath = config.get(BUCKET_PATH_FIELD).asText();
+      final String customOutputFormat = String.join("/",
```

Comment:
does `customOutputFormat` still make sense as the name here when we're including bucketPath?

Reply:
you can still customize the end part of the bucket path

it just always starts with the bucketPath now